src/registers/cpuectrl_el1.rs

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Copyright (c) 2018-2022 by the author(s)
+//
+// Author(s):
+//   - Valentin B. <valentin.be@protonmail.com>
+
+//! CPU Extended Control Register - EL1
+//!
+//! Provides additional implementation-defined configuration and control options
+//! for the core.
+
+use tock_registers::{
+    interfaces::{Readable, Writeable},
+    register_bitfields,
+};
+
+register_bitfields! {u64,
+    pub CPUECTRL_EL1 [
```

Comment:
This doesn't seem to be defined in the ARMv8 ARM?

Reply:
Oops, it seems I mixed in a model-specific register with this one. Reverted.